src/Microsoft.AspNet.Mvc.ApiExplorer/project.json

```diff
@@ -0,0 +1,19 @@
+﻿{
+  "description": "The API explorer component for ASP.NET MVC.",
```

Comment:
@Eilon, @rynowak : Please let me know what you think of this description.

"The ASP.NET MVC's API explorer component for discovering metadata like http method, relative path etc for actions on controllers."